_docs/examples/macos.md

```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
 You may also need to disable [System Integrity Protection](https://support.apple.com/en-us/HT204899).
 
 _Please click "Improve this page" above and add an example. Thanks!_
+
+{% highlight py %}
+
+import frida
+import sys
+
+def on_message(message, data):
+    print("[%s] => %s" % (message, data))
+
+def main(target_process):
+    session = frida.attach(target_process)
+
+    script = session.create_script("""
+	// Get a reference to the openURL selector
```

Comment:
- This is `Get a reference to the openURL selector` => `Get a reference to the openURL method`.
- Indentation seems off from the next line.